shell/lisa_shell

```diff
@@ -219,6 +219,34 @@ function _lisa-install-system-check {
     _lisa-python-package-check ensurepip python3-venv
 }
 
+function _lisa-copy-git-hook {
+     # works even with worktrees
+     local hooks_dir="$(git -C "$LISA_HOME" rev-parse --git-common-dir)/hooks"
+     local hook="$1"
+
+     # Copy/Update lisa specific git-hook
+     cp "$LISA_HOME/tools/$hook.lisa" "$hooks_dir/$hook.lisa"
+
+     # Avoid overriding user hooks
+     if [[ ! -e "$hooks_dir/$hook" ]]; then
+         cp "$LISA_HOME/tools/$hook" "$hooks_dir/$hook"
+     else
+         installed=`grep -qxF '"$0".lisa' "$hooks_dir/$hook" || echo $?`
+
+         if [[ $installed -ne 0 ]]; then
```

Comment:
Can be simplified into:
```

     if [[ ! -e "$hooks_dir/$hook" ]]; then
         cp "$LISA_HOME/tools/$hook" "$hooks_dir/$hook"
     elif ! grep -qxF '"$0".lisa' "$hooks_dir/$hook"; then
         echo ""
         echo -ne "$LISASHELL_RED"
         echo "!!WARNING!! Couldn't copy git-hook: $LISA_HOME/tools/$hook"
         echo -ne "$LISASHELL_RESET"
    fi
```

Reply:
Does it make a difference to warrant the churn?

Reply:
There's no concerning churn, the PR is not merged yet. In any case, a PR update would be needed to make `installed` variable local, so we might as well eliminate it.

Also I've seen this pattern with multiplication of variables and tests in a lot of the existing code in LISA so I assume people don't know `if` takes a command as parameter, hence the comments. Next time there should not be a need for such comment hopefully.

Reply:
Okay I converted install to local. I'm aware of if can take a command but I don't like this style in general. Variables are there to help code more readable, and my preference is to keep it like this because you can still skim the code and understand what is happening without having to read the whole grep command.